feature-configs/cn-ran-overlays/all-in-one/cluster/kustomization.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  # PTP operator deployment
+  - deployment/ptp
+  # PTP profiles
+  - du-common/ptp
+
+  # SR-IOV deployment (Networks are deployed as a part of site configuration)
+  - deployment/sriov
+  # SR-IOV network node policies
+  - ../example/sriov
+
+  # SCTP deployment (There is no profile)
+  - deployment/sctp
+
+  # PAO deployment
+  - deployment/performance
+  # PAO profile is given as an example. Must be adapted for a specific hardware
+  - ../example/performance
```

Comment:
we have an example profile
https://github.com/openshift-kni/cnf-features-deploy/blob/master/feature-configs/typical-baremetal/performance/performance_profile.patch.yaml

Reply:
For customer documentation, to need to keep the RAN profiles in a single space.  This will make the description of how to deploy a CU/DU manually much simpler.  Adding in a separate example adds indirection.

Reply:
There is a difference between the profiles, and we didn't want to interfere.

Reply:
+1 on keeping the ran specific configurations explicit and separated.